forest/cli/src/cli/mod.rs

```diff
@@ -14,12 +14,12 @@ mod db_cmd;
 mod fetch_params_cmd;
 mod mpool_cmd;
 mod net_cmd;
-mod send_cmd;
+pub mod send_cmd;
 mod shutdown_cmd;
 mod snapshot_cmd;
 mod state_cmd;
 mod sync_cmd;
-mod wallet_cmd;
+pub mod wallet_cmd;
```

Comment:
These modules shouldn't be public. Instead of using public modules, you can move your tests into the modules.